lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -61,11 +66,21 @@ def _create_db_locations(self, db_location: str, n_shards: int) \
             )
             logging.info("db location: %s", db_location_list[shard])
 
+        return db_location_list
+
+    def _create_digest_db_locations(self, db_location: str) -> [str, str]:
         digest_db_location = 'file:' + db_location + \
                              'subscriber-digest.db?cache=shared'
         logging.info("digest db location: %s", digest_db_location)
 
-        return db_location_list, digest_db_location
+        per_sub_digest_db_location = 'file:' + db_location + \
```

Comment:
Maybe I'm missing something here, but what is the difference between `per_sub_digest_db` and `digest_db`? afaik `digest_db` is used to store the flat digests right? Isn't that per subscriber as well?

Reply:
`digest_db` is used to store exclusively *flat digests*, which are digests for an entire network (including all subscribers in it);
`per_sub_digest_db` stores exclusively *per-subscriber digests*, which are digests for every individual subscriber for every network. Hope this helps! @ardzoht